code/modules/judaism/foreskin.dm

```diff
@@ -0,0 +1,69 @@
+/obj/item/organ/foreskin
+	name = "foreskin"
+	desc = "A small strip of skin harvested from a male. Certain cultures attribute mythical powers to these."
+	zone = BODY_ZONE_PRECISE_GROIN
+	slot = ORGAN_SLOT_FORESKIN
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "scrap"
+	var/foreskincolor = "#ffe0d1"
+
+/obj/item/organ/foreskin/Insert(mob/living/carbon/M)
+	..()
+
+	if (M)
+		if (M.gender == FEMALE)
+			qdel(src)
+
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		foreskincolor = "#" + skintone2hex(H.skin_tone)
+	add_atom_colour(foreskincolor, FIXED_COLOUR_PRIORITY)
+
+/obj/item/organ/foreskin/prepare_eat()
+	var/obj/item/reagent_containers/food/snacks/organ/S = new
+	S.name = name
+	S.desc = desc
+	S.icon = icon
+	S.icon_state = icon_state
+	S.w_class = w_class
+	S.add_atom_colour(foreskincolor, FIXED_COLOUR_PRIORITY)
+
+	return S
+
+/datum/surgery_step/incise/circumcise
+	name = "circumcise"
+	var/obj/item/organ/foreskin/F = null
+
+/datum/surgery/circumcision
```

Comment:
I'd suggest a "uncircumcise" surgery to revert the effect, Make it require some heavy weights as a surgical tool to stretch the remaining skin.